http/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "http"
```

We probably need to name this something less generic like `rust-ipfs-http` or some such thing because of https://crates.io/crates/http

---

```suggestion
name = "rust-ipfs-http"
```

This would be in the similar fashion as other subcrates I've seen, like for example rust-libp2p.